v2/pkg/mongo-driver/main_test.go

```diff
@@ -0,0 +1,70 @@
+package mongo_driver_test
+
+import (
+	"flag"
+	"os"
+	"testing"
+
+	componentTest "github.com/ONSdigital/dp-component-test"
+	"github.com/cucumber/godog"
+	"github.com/cucumber/godog/colors"
+)
+
+const MongoVersion = "4.0.23"
+const MongoPort = 27017
+const DatabaseName = "testing"
+const CollectionName = "testCollection"
+
+type MongoDBComponentTest struct {
+	MongoFeature     *componentTest.MongoFeature
+	MongoV2Component *MongoV2Component
+}
+
+var componentFlag = flag.Bool("component", false, "perform component tests")
+
+func (m *MongoDBComponentTest) InitializeTestSuite(ctx *godog.TestSuiteContext) {
+	ctx.BeforeSuite(func() {
+		m.MongoFeature = componentTest.NewMongoFeature(componentTest.MongoOptions{MongoVersion: MongoVersion, DatabaseName: DatabaseName})
+		m.MongoV2Component = newMongoV2Component(DatabaseName, CollectionName, m.MongoFeature.Client)
+	})
+	ctx.AfterSuite(func() {
+		m.MongoFeature.Close()
+	})
+}
+
+func (m *MongoDBComponentTest) InitializeScenario(ctx *godog.ScenarioContext) {
+
```

Comment:
I'm not going to ask you to go through and change them all but something I've noticed across the ONS.. there seems to be no consistency to when blank lines are added at the beginning of functions, probably by a 'case by case as it feels' basis.. And I understand I used to do exactly the same, but after working through a linter which flagged this as an error for quite some time I've come to agree with the linter. 
If nothing else consistency is always preferred, and there's no real need to add blank lines.
I suspect it's something that's been picked up from one of the testing packages, I feel as a rule (especially in non-testing code) we should not use blank lines at the beginning of functions (with talk of linters being introduced soon this may end up happening anyway)

Reply:
I have updated the file to be more consistent